shared/params/mainnet_config.go

```diff
@@ -123,6 +123,7 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	MaxPeersToSync:            15,
 	SlotsPerArchivedPoint:     2048,
 	GenesisCountdownInterval:  time.Minute,
+	HeadOfVotingPeriodLength:  32,
```

Comment:
`HeadOfVotingPeriodLength` can easily be confused as one of the beacon chain config. 
I suggest we add `eth1` to the name

Reply:
+1 it conflicts with a well known param.

Reply:
This parameter has been removed. See Preston's comment for more details.